frontend/messages/en.json

```
@@ -776,6 +776,10 @@
 	"startDate": "Start date",
 	"startDateHelpText": "Start date (useful for timeline)",
 	"backupLoadingError": "An error occurred while loading the backup.",
+	"backupVersionError": "Can't load the backup, the version of the backup is invalid",
+	"confirmYes": "Confirm by typing 'yes' below:",
+	"confirmYesPlaceHolder": "Type 'yes' to confirm",
+	"yes": "yes",
```

Biome (1.9.4)

[error] The key yes was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)